pages/_app.tsx

```diff
@@ -36,6 +37,7 @@ function MyApp({ Component, pageProps }: AppProps): React.ReactElement {
         }}
       >
         <Component {...pageProps} />
+        <Script src="https://paypal.github.io/skipto/downloads/js/skipto.min.js" />
```

Comment:
@yonikid15 Including this library from a third-party CDN is not a good idea for [several reasons](https://httptoolkit.tech/blog/public-cdn-risks/) including reliability and security. There is an [npm package](https://www.npmjs.com/package/skipto) available but it is not written in TypeScript and not written in module syntax for importing.

Reply:
> Including this library from a third-party CDN is not a good idea for [several reasons](https://httptoolkit.tech/blog/public-cdn-risks/) including reliability and security. There is an [npm package](https://www.npmjs.com/package/skipto) available but it is not written in TypeScript and not written in module syntax for importing.

Please see previous comments, and [this issue](https://github.com/paypal/skipto/issues/120) that speaks to the potential changes afoot with the library and maintainers. Is implementing this library a requirement for this project?